meya-customers/fitness-demo
# Strava use-case

[Miro](https://miro.com/app/board/o9J_lPWHcA0=/)
[Loom](https://www.loom.com/share/f632ce7b8f50403e92cf2c82bcbd61e8)
[Codepen](https://codepen.io/ekalvi/pen/dyNrmaQ)

# Testing
* Open the Codepen. Click the **Privacy Settings** button. 
* Confirm that Orb opens. After going through the flow, try viewing the help center article.
* Click the **Connect Device** button. You can use Zwift or Wahoo.
* In the Zwift or Wahoo flow, try escalating to an agent. If `isPaid` is `true` in `pageContext`, the user should be directed to ZDC. If `false`, they should be directed to ZDS.
